src/util/path.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as os from 'os';
+
+/**
+ * Get the base path based on a path
+ */
+export function AbsolutePathFromRelativePath(partialPath: string): string {
```

Comment:
Can you rename this to...`GetAbsolutePath` and adjust the comment? should probably say something more along the lines of:
`Given relative path, calculate absolute path`.